code/game/objects/items/ego_weapons/non_abnormality/liu.dm

```diff
@@ -84,3 +83,156 @@
 							JUSTICE_ATTRIBUTE = 100
 							)
 
+
+
+//Section 4/5/6, 6-4
+/obj/item/ego_weapon/city/liu/fist
+	name = "liu combat gloves"
+	icon_state = "liufist"
+	force = 20
+	attack_speed = 0.7
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 60,
+							PRUDENCE_ATTRIBUTE = 40,
+							TEMPERANCE_ATTRIBUTE = 40,
+							JUSTICE_ATTRIBUTE = 40
+							)
+	var/chain = 0
+	var/activated
+	hitsound = 'sound/weapons/fixer/generic/fist1.ogg'
+
+/obj/item/ego_weapon/city/liu/fist/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>This weapon has light and heavy attacks. Use in hand to activate a heavy attack. Combos are as follows:"
+	. += "<span class='notice'>LLLLL - 5 Hit fast combo, ending in a knockback attack."
+	. += "<span class='notice'>H 	 - Windup fist attack for 1.5x damage and deals massive stamina damage to humans."
+	. += "<span class='notice'>LH 	 - AOE Fire fist attack. This does not kill insane people."
+	. += "<span class='notice'>LLH 	 - High Damage Combo, last attack has a windup and deals 2x damage."
+	. += "<span class='notice'>LLLH  - Deals good damage. Last hit backsteps you 2 tiles."
+	. += "<span class='notice'>LLLLH - High Damage combo, last hit ends in a 2x damage boost with no windup."
```

Comment:
```suggestion
	. += "<span class='notice'>This weapon has light and heavy attacks. Use in hand to activate a heavy attack. Combos are as follows:</span>"
	. += "<span class='notice'>LLLLL - 5 Hit fast combo, ending in a knockback attack.</span>"
	. += "<span class='notice'>H 	 - Windup fist attack for 1.5x damage and deals massive stamina damage to humans.</span>"
	. += "<span class='notice'>LH 	 - AOE Fire fist attack. This does not kill insane people.</span>"
	. += "<span class='notice'>LLH 	 - High Damage Combo, last attack has a windup and deals 2x damage.</span>"
	. += "<span class='notice'>LLLH  - Deals good damage. Last hit backsteps you 2 tiles.</span>"
	. += "<span class='notice'>LLLLH - High Damage combo, last hit ends in a 2x damage boost with no windup.</span>"
```

Reply:
It works as written btw, just a heads up